aimmo-game/simulation/maps/cell.py

```diff
@@ -0,0 +1,58 @@
+from simulation.action import MoveAction
+
+
+class Cell(object):
+    """
+    Any position on the world grid.
+    """
+
+    def __init__(self, location, habitable=True, generates_score=False, partially_fogged=False):
+        self.location = location
+        self.habitable = habitable
+        self.generates_score = generates_score
```

Comment:
so are we keeping this for the moment?

Reply:
Nope. Doing the change now!

Reply:
Actually, nevermind. It's not as easy as that @CelineBoudier - will need another PR if that's fine.

Reply:
#248